runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/Produce.java

```diff
@@ -102,17 +105,31 @@
 	PINEAPPLE("Pineapple", ItemID.PINEAPPLE, 160, 7, 45, 7),
 	PAPAYA("Papaya", ItemID.PAPAYA_FRUIT, 160, 7, 45, 7),
 	PALM("Palm", ItemID.COCONUT, 160, 7, 45, 7),
+	DRAGONFRUIT("Dragonfruit", ItemID.DRAGONFRUIT, 160, 7, 45, 7),
+
+	// Cactus
+	CACTUS("Cactus", ItemID.CACTUS_SPINE, 80, 8, 20, 4),
+	POTATO_CACTUS("Potato cactus", ItemID.POTATO_CACTUS, 10, 8, 5, 7),
+
+	// Hardwood
+	TEAK("Teak", ItemID.TEAK_LOGS, 560, 8),
+	MAHOGANY("Mahogany", ItemID.MAHOGANY_LOGS, 640, 9),
+
+	// Anima
+	ATTAS("Attas", NullItemID.NULL_22940, 640, 9),
+	IASOR("Iasro", NullItemID.NULL_22939, 640, 9),
+	KRONOS("Kronos", NullItemID.NULL_22938, 640, 9),
 
 	// Special crops
 	SEAWEED("Seaweed", ItemID.GIANT_SEAWEED, 10, 5, 0, 4),
-	TEAK("Teak", ItemID.TEAK_LOGS, 560, 8),
 	GRAPE("Grape", ItemID.GRAPES, 5, 8, 0, 5),
 	MUSHROOM("Mushroom", ItemID.MUSHROOM, 40, 7, 0, 7),
-	MAHOGANY("Mahogany", ItemID.MAHOGANY_LOGS, 640, 9),
-	CACTUS("Cactus", ItemID.POTATO_CACTUS, 80, 8, 20, 4),
 	BELLADONNA("Belladonna", ItemID.CAVE_NIGHTSHADE, 80, 5),
 	CALQUAT("Calquat", ItemID.CALQUAT_FRUIT, 160, 9, 0, 7),
-	SPIRIT_TREE("Spirit Tree", ItemID.SPIRIT_TREE, 320, 13);
+	SPIRIT_TREE("Spirit tree", ItemID.SPIRIT_TREE, 320, 13),
+	CELASTRUS("Celastrus", ItemID.BATTLESTAFF, 160, 6, 0, 4),
```

Comment:
```suggestion
	CELASTRUS("Celastrus", ItemID.CELASTRUS_BARK, 160, 6, 0, 4),
```

This should be the bark

Reply:
That's what the tree produces, and it looks better imo

Reply:
A battlestaff is more recognizable. Note that this is just the icon, it does nothing else.